Publish sponsorship tiers (#266)
servo/servo.org: sponsorship.md

</div>

<div id="sponsorship-tiers">

## Sponsorship Tiers

- **Platinum: 10,000 USD/month**
- **Gold: 5,000 USD/month**
- **Silver: 1,000 USD/month**
- **Bronze: 100 USD/month**

> By sponsoring through one of the tiers **your logo/name will be displayed at [servo.org](https://servo.org/)**, provided that it doesn't violate [Servo's Code of Conduct](https://servo.org/coc).

Reviewer: Maybe we can rewrite this to make it sound less transactional:

```suggestion
Sponsors in the Bronze tier and above will be acknowledged on [servo.org](https://servo.org/) with **their logo or name**.
```

Not sure if we want to mention the CoC as the TSC will review anyway and look at more than just the CoC.

<p class="subtitle">
  If you are interested in sponsoring the Servo project through one of this tiers, please email us at <a href="mailto:join@servo.org">join@servo.org</a>.
</p>

> A few clarifications:

Reviewer: Not sure if we still need to list these disclaimers. Might be more welcoming and less transactional to just say "please contact us". If we want to clarify that rejections are a possibility, we can amend the previous sentence to say

"If you are interested in sponsoring the Servo project through one of these tiers, please contact us at <a href="mailto:join@servo.org">join@servo.org</a>. Sponsorship requests for these tiers will be reviewed by the TSC."

Reviewer: I'm fine if we want to leave these in.

Reviewer 2: I think since we now agreed to request people contact us, we could remove the clarifications (e.g. I could see us committing to something for a platinum sponsor, if it aligns with our goals, the right to reject is implied by asking people to contact us, and non-endorsement makes less sense if the TSC approved the sponsor)
